blocks/results-list-block/_index.scss

```
@@ -0,0 +1,15 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-result-list {
```

Minor thing - the block is plural - can we make the styles match too?

```suggestion
.b-results-list {
```